mq/main/comm-util/src/main/java/com/sun/messaging/jmq/util/DestType.java

```diff
@@ -112,10 +113,7 @@ public static boolean isQueueStr(String queueOrTopic) {
         if (queueOrTopic == null) { // should never happen
             return false;
         }
-        if (queueOrTopic.equals(QUEUESTR)) {
-            return true;
-        }
-        return false;
+        return queueOrTopic.equals(QUEUESTR);
```

Comment:
If changing this - perhaps
```java
return Objects.equals(QUEUESTR, queueOrTopic);
```
could do? That way the first condition on `null` could be removed too.